Stored ID handling - Phase 2 (#1309)
10up/distributor: docs/stored-id-handling.md

      - `block_name` (`string`): The block name. **(Required)**
      - `block_attribute` (`string|array`): The attribute(s) containing the data. **(Required)**

  - **`type`** (`string`):

>     Type of data. Accepted values are `'media'`, `'post'`, or `'term'`. If set, default callbacks can be used.

Reviewer: ```suggestion
    Type of data. Accepted values are `'media'`, `'post'`, or `'term'`. If set, the default callbacks will be used.
```

>     **Note:** This parameter cannot be used in conjunction with custom callbacks. If custom callbacks are provided, they will take precedence over the default behavior.

Reviewer: If I'm following the code correctly, looks like we log an error if this parameter is used and custom callbacks are set. This may be something we just silently ignore if someone has this set up that way but it seems this comment isn't accurate as things will work if both are set
```suggestion
    **Note:** If this parameter is used in conjunction with custom callbacks, the custom callbacks will be used instead.
```
